test/Main.purs

```diff
@@ -43,23 +43,23 @@ derive instance genericLiteralStringExample :: Generic LiteralStringExample _
 instance showLiteralStringExample :: Show LiteralStringExample where
   show a = genericShow a
 instance encodeJsonLiteralStringExample :: EncodeJson LiteralStringExample where
-  encodeJson a = encodeLiteralSumWithTransform id a
+  encodeJson a = encodeLiteralSumWithTransform (\x->x) a
```

Comment:
Why replace id with a hand-written id?

Reply:
ah yes - thanks for reminding me, I wanted to check that out

for some reason `id` was not in scope anymore and instead of trying to add yet another yak on my shaving stack I opted to just put it in myself

I'll try to check this out tomorrow

Reply:
id -> identity

Reply:
ah I see - that's `identity` from `Control.Category` right?

Reply:
it's in prelude anyway

Reply:
yeah ide-server figured that out for me ;)

I understand the reasoning but it's really hard to find using pursuit when you look for the usual `a -> a`

Reply:
that's why there's typed hole search: https://github.com/paf31/24-days-of-purescript-2016/blob/master/23.markdown

Reply:
fair enough I guess (I'm probably not using that feature enough)